### PR TITLE
Feature/appc 964 p2p tranfer

### PR DESCRIPTION
**Pull request by Trinkes**

### CODE REVIEW

File: app/src/main/res/layout/transact_fragment_layout.xml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background_card"
```
Comment (JoaoRaimundo) [inline app/src/main/res/layout/transact_fragment_layout.xml:6]: Add `android:fillViewport="true"` to make sure that the scroll view uses the full screen even when it's not needed.

File: app/src/main/res/layout/transact_fragment_layout.xml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        app:layout_constraintBottom_toTopOf="@id/send_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/currency_selector"
+        tools:text="Current balance: 867 APPC-C"
+        />
+    <Button
+        android:id="@+id/send_button"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
```
Comment (JoaoRaimundo) [inline app/src/main/res/layout/transact_fragment_layout.xml]: Set the button height to 32dp

File: app/src/main/java/com/asfoundation/wallet/ui/transact/TransactPresenter.kt [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    handleQrCodeResult()
+    handleCurrencyChange()
+  }
+
+  private fun handleCurrencyChange() {
+    disposables.add(view.getCurrencyChange()
+        .subscribeOn(viewScheduler)
+        .observeOn(ioScheduler)
+        .switchMapSingle {
+          getBalance(it).map { balance -> BalanceUtils.weiToEth(balance) }
```
Comment (JoaoRaimundo) [inline app/src/main/java/com/asfoundation/wallet/ui/transact/TransactPresenter.kt]: APPC and ETH balance don't require this conversion (weiToEth)

File: app/src/main/res/layout/transact_fragment_layout.xml (hunk 1 above)
Comment (Trinkes) [reply app/src/main/res/layout/transact_fragment_layout.xml:6]: can we leave it to other release?

**Pull request merged**